core/src/subgraph/instance_manager.rs

```diff
@@ -929,6 +929,22 @@ where
         info!(&logger, "Applying {} entity operation(s)", mods.len());
     }
 
+    if !block_state.deterministic_errors.is_empty() {
+        info!(
```

Comment:
Does it make sense to log this at info? Maybe when you log the individual errors, have something like `Subgraph error (3/17)` in the log message.

Reply:
That is better I've updated the PR. Also removed the `id`, it's implicit in the logger context.